### PR TITLE
Dennis feature1

### PR DESCRIPTION
**Pull request by QA-ArsalanAsad**

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a file
```
Comment (jnfernQA) [inline file1.txt:1]: Looks good

**Review by jnfernQA — CHANGES_REQUESTED** [review]
nice

**Review by jnfernQA — APPROVED** [review]
nice

File: file1.txt (hunk 1 above)
Comment (MuratOzturk18) [inline file1.txt:1]: I like it.

**Review by MuratOzturk18 — APPROVED** [review]

**Review by DDuncan-qa — APPROVED** [review]
Great!

**Review by aktarshahnazqa — APPROVED** [review]

**Comment by QA-ArsalanAsad** [comment]
finished

**Pull request merged**